src/app/modules/form/admin-form/admin-form.service.ts

```diff
@@ -1177,8 +1179,16 @@ const isMobileFieldUpdateAllowed = (
  * @returns string representing the msgSrvcName
  */
 // Export for testing
-export const generateMsgSrvcName = (formId: string) =>
-  `formsg/${config.secretEnv}/form/${formId}/twilio`
+export const generateMsgSrvcName = (formId: string) => {
+  const res = `formsg/${config.secretEnv}/api/${formId}/twilio/${uuidv4()}`
+  console.log(res)
+  return res
+}
+
+export const isApiGenerated = (msgSrvcName: string) => {
```

Comment:
nit: should we rename this function to better reflect what it is doing? Suggest `checkIsApiSecretKeyName`

Reply:
please also add JSDocs to both functions